uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/tab/TabPanelEntry.java

```diff
@@ -157,5 +165,21 @@ public HandlerRegistration addMouseDownHandler(MouseDownHandler handler) {
             return addDomHandler(handler,
                                  MouseDownEvent.getType());
         }
+
+        @Override
+        public OuiaComponentTypeAttribute ouiaComponentType() {
+            return new OuiaComponentTypeAttribute("editor-nav-tab");
+        }
+
+        @Override
+        public OuiaComponentIdAttribute ouiaComponentId() {
+            return new OuiaComponentIdAttribute("editor-nav-tab-" + getText());
```

Comment:
Just FYI:
We don't need that prefix, because id is supposed to just distinguish between components of same component-type and on the same level only - it's not supposed to be globally unique across various component types. In other words, it's perfectly fine to have multiple elements same value for data-ouia-component-id if they are either:
* on different levels of DOM tree
* of different data-ouia-component-type

Reply:
Will simplify the id, thank you